run.py

```diff
@@ -33,6 +33,20 @@
     help="Do not drop last validate batch if "
     "if it is smaller than batch_size.",
 )
+@click.option(
+    "--affinity",
+    "-af",
+    type=str,
+    default=None,
+    help="Path to affinity matrix for training.",
```

Comment:
Same as comment above, need to be consistent if is a path or filename.

Reply:
Sounds good, I'll patch to remove the legacy os.path.join with datapath and so this will actually use a full path for consistency with classes.